torch/csrc/jit/passes/onnx/preinline_onnx.cpp

```diff
@@ -0,0 +1,61 @@
+#include <torch/csrc/jit/passes/onnx/preinline_onnx.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+
+namespace torch {
+namespace jit {
+
+const auto namespaceName = std::string("___torch_mangle_3");
```

Comment:
Just curious - is there a convention for picking these namspaceName?

Reply:
That's a question I wanted to ask @eellison since I see the namespace can change. I'll have to check this.